app/models/route.rb

```diff
@@ -3,4 +3,15 @@ class Route < ApplicationRecord
   has_many :bus_stops_routes
   has_many :bus_stops, through: :bus_stops_routes
   default_scope { order :number }
+
+  def stops_by_sequence(direction)
+    bus_stops.where(bus_stops_routes: { direction: direction })
+             .order('bus_stops_routes.sequence')
```

Comment:
OOC, does this not work as just `order(:sequence)`? I think BSR is the only table with a `sequence` column.

Reply:
Huh, who knew?  From the Rails 5 documentation:

```ruby
User.order(:name) # SELECT "users".* FROM "users" ORDER BY "users"."name" ASC
User.order('name') # SELECT "users".* FROM "users" ORDER BY name
```